demos/checkbox.scss

```diff
@@ -17,4 +17,39 @@
 @import "./common";
 @import "../packages/mdc-checkbox/mdc-checkbox";
 @import "../packages/mdc-form-field/mdc-form-field";
+@import "../packages/mdc-theme/color_palette";
+@import "../packages/mdc-theme/mixins";
 @import "../packages/mdc-typography/mdc-typography";
+
+.mdc-checkbox.demo-checkbox--custom-stroke-and-fill {
+  $color: $material-color-red-500;
+
+  @include mdc-checkbox-focus-indicator-color($color);
+  @include mdc-checkbox-ripple((base-color: $color, opacity: $mdc-checkbox-ripple-opacity));
+  @include mdc-checkbox-stroke-and-container-fill-color(
+    $unmarked-stroke-color: $color,
+    $unmarked-container-fill-color: rgba($color, .25),
```

Comment:
Should this .25 be a public variable in mdc-checkbox?

Reply:
IMHO probably not - this is just a demo page, and we don't have any official material guidelines about what opacity to use for fill color of an unmarked checkbox.